### PR TITLE
fix: bind configs to variables before exporting to make sure typeerrors are reported

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

Errors weren't reported when they were exported immediately:

CleanShot 2023-02-21 at 10 30 56@2x

Binding it to a variable before exporting fixes it

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: cli/template/extras/config/postcss.config.cjs [hunk 1]
```diff
@@ -1,6 +1,8 @@
-module.exports = {
+const config = {
```
Comment (juliusmarminge) [inline cli/template/extras/config/postcss.config.cjs:1]: dont think there is a type we can use here but oh well - might as well be consistent

Comment (bai) [reply cli/template/extras/config/postcss.config.cjs:1]: Wonder if `/** @type {import('postcss-load-config').Config */` would work:

* https://github.com/matrix-org/matrix-authentication-service/blob/64ce271d08a8b38f03ff0317d2f04dd69c264f5f/frontend/postcss.config.cjs#L15
* https://github.com/vnphanquang/svelte-put/blob/e625fe8a794b8aee4aa811d960ea0448d686cb5d/sites/docs/postcss.config.cjs#L5

File: .changeset/odd-spies-appear.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"create-t3-app": patch
+"@ct3a/www": patch
```
Comment (juliusmarminge) [inline .changeset/odd-spies-appear.md:3]:
```suggestion
```

**Comment by juliusmarminge** [comment]
@nexxeln @c-ehrlich approval please

**Review by c-ehrlich — APPROVED** [review]

**Pull request merged**